### ISSUE TITLE
User defined connection profiles not deletable

### ISSUE THREAD

**Issue by satkunas** [inverse-inc/packetfence#3917]
The 2nd item in the following array should have: `not_deletable: false`

Taken from 

**/api/v1/config/connection_profiles**
```
{
    "items": [{
        "access_registration_when_registered": "disabled",
        "always_use_redirecturl": "disabled",
        "autoregister": "disabled",
        "billing_tiers": [],
        "block_interval": {
            "interval": "10",
            "unit": "m"
        },
        "default_psk_key": null,
        "description": "Default Profile",
        "device_registration": "",
        "dot1x_recompute_role_from_portal": "enabled",
        "dpsk": "disabled",
        "id": "default",
        "locale": ["en_US", "fr_CA", "fr_FR", "es_ES"],
        "login_attempt_limit": "0",
        "logo": "\/common\/packetfence-cp.png",
        "network_logoff": "disabled",
        "network_logoff_popup": "disabled",
        "not_deletable": true,
        "preregistration": "disabled",
        "provisioners": [],
        "redirecturl": "http:\/\/www.packetfence.org\/",
        "reuse_dot1x_credentials": "0",
        "root_module": "default_pending_policy",
        "scans": [],
        "sms_pin_retry_limit": "0",
        "sms_request_limit": "0",
        "sources": [],
        "status": "enabled",
        "unreg_on_acct_stop": "disabled"
    }, {
        "access_registration_when_registered": "disabled",
        "advanced_filter": null,
        "always_use_redirecturl": "disabled",
        "autoregister": "disabled",
        "billing_tiers": [],
        "block_interval": {
            "interval": "10",
            "unit": "m"
        },
        "default_psk_key": null,
        "description": "Default Profile",
        "device_registration": "",
        "dot1x_recompute_role_from_portal": "enabled",
        "dpsk": "disabled",
        "filter": [{
            "match": "default",
            "type": "node_role"
        }],
        "filter_match_style": "all",
        "id": "default2",
        "locale": ["en_US", "fr_CA", "fr_FR", "es_ES"],
        "login_attempt_limit": "0",
        "logo": "\/common\/packetfence-cp.png",
        "network_logoff": "disabled",
        "network_logoff_popup": "disabled",
        "not_deletable": true,
        "preregistration": "disabled",
        "provisioners": [],
        "redirecturl": "http:\/\/www.packetfence.org\/",
        "reuse_dot1x_credentials": "0",
        "root_module": "default_pending_policy",
        "scans": [],
        "sms_pin_retry_limit": "0",
        "sms_request_limit": "0",
        "sources": [],
        "status": "enabled",
        "unreg_on_acct_stop": "disabled"
    }],
    "nextCursor": 2,
    "prevCursor": 0,
    "status": 200
}
```

**Comment by jrouzierinverse** [comment]
Fixed by d37c51a6afa4068776922fbf950662b1ea4f3933

**Comment by satkunas** [comment]
Problem still persists
